src/viking.js

```diff
@@ -1,11 +1,53 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength){
+    this.health = health;
+    this.strength = strength;
+  }
+  attack(){
+    return this.strength;
+  }
+  receiveDamage(damage){
+    return this.health -= damage
+  }  
```

Comment:
According to instructions, receiveDamage() method inside Soldier class should not return anything - only calculate the result

```
  receiveDamage(damage){
    this.health -= damage
  }
```